autoload/db.vim

```diff
@@ -11,6 +11,15 @@ function! s:expand(expr) abort
   return exists('*DotenvExpand') ? DotenvExpand(a:expr) : expand(a:expr)
 endfunction
 
+" Hide pointless `No matching autocommands` in Vim
+if !has('nvim')
```

Comment:
Let's ditch this conditional. I'd rather have a technically unnecessary autocommand on nvim than a subtle behavior change between vim and nvim.

Reply:
Fixed.